che-theia-plugins.yaml

```diff
@@ -406,7 +406,7 @@ plugins:
       - https://github.com/SonarSource/sonarlint-vscode/releases/download/1.16.0/sonarlint-vscode-1.16.0.vsix
   - repository:
       url: https://bitbucket.org/atlassianlabs/atlascode
-      revision: 2.6.6
+      revision: 2.8.5
     extensions:
       - https://download.jboss.org/jbosstools/vscode/3rdparty/atlascode/atlascode-2.6.6.vsix
```

Comment:
Vsix should be updated as well

Reply:
@matthewfisch do you have a link to a built vsix somewhere?

Reply:
@ericwill Took me a while to get Atlassian to publish outside the microsoft marketplace. This is in openvsx now:

https://open-vsx.org/api/atlassian/atlascode/2.8.6/file/atlassian.atlascode-2.8.6.vsix. I am abandoning this PR will open a new one.